test/cql-pytest/test_materialized_view.py

```diff
@@ -808,3 +808,14 @@ def test_mv_with_only_primary_key_rows(scylla_only, cql, test_keyspace):
             nodetool.flush(cql, view)
             assert(set([row.id for row in cql.execute(f'SELECT id FROM {view}')]) == set([1, 2, 3]))
             # We now believe that empty value serialization/deserialization is correct
+
+def test_mv_prepared_statement_with_altered_base(scylla_only, cql, test_keyspace):
```

Comment:
Please add a comment what this test checks, not just in the commit message but also before the actual test. Most importantly it should please mention that it reproduces issue #16392. This information will be very useful when we come across a failing test and don't remember what it was about, or what we forgot to backport.

Reply:
Also, is this test really "scylla_only" - meaning it can't run on Cassandra? Why?
If it's *really* scylla_only please explain why in the commit message.